### PR TITLE
add eslint with this configurations

### PR DESCRIPTION
**Pull request by AbdilFattahBoshi**
Hii @NourSammour  I have added eslint with its configurations kindly review it

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1362 @@
+{
```
Comment (NourSammour) [inline package-lock.json]: please remove this file

**Review by NourSammour — CHANGES_REQUESTED** [review]
Please fix the conflicts

File: package-lock.json (hunk 1 above)
Comment (AbdilFattahBoshi) [reply package-lock.json]: Ok

**Comment by AbdilFattahBoshi** [comment]
I have resolve the conflicts and delete package-lock.json file 
please review it again @NourSammour

File: package.json [hunk 2]
```diff
@@ -2,6 +2,18 @@
   "name": "biletiniz",
   "version": "1.0.0",
   "description": "Uygun uçak bileti Nereye isterseniz | Biletiniz.com",
+  "main": "index.js",
+  "scripts": {},
```
Comment (NourSammour) [inline package.json]: why this? we have scripts tag in line 17
same for `main`, `bugs`, `homepage`

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Review by NourSammour — CHANGES_REQUESTED** [review]
ignore `.next` folder

**Pull request closed without merging**